<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="Hanifes-MBP-2.home" name="com.qa.hotspot.tests.HomePageTest" tests="4" failures="0" timestamp="2020-07-23T19:11:45 EDT" time="0.013" errors="1">
  <testcase name="setUp" time="0.013" classname="com.qa.hotspot.tests.HomePageTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.hubspot.base.BasePage.init_driver(BasePage.java:26)
at com.qa.hotspot.tests.HomePageTest.setUp(HomePageTest.java:29)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:64)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:364)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:318)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:620)
at org.testng.TestRunner.beforeRun(TestRunner.java:610)
at org.testng.TestRunner.run(TestRunner.java:581)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setUp -->
  <system-out/>
  <testcase name="verifyHomePageTitle" time="0.000" classname="com.qa.hotspot.tests.HomePageTest">
    <skipped/>
  </testcase> <!-- verifyHomePageTitle -->
  <system-out/>
  <testcase name="verifyHomePageHeaderTest" time="0.000" classname="com.qa.hotspot.tests.HomePageTest">
    <skipped/>
  </testcase> <!-- verifyHomePageHeaderTest -->
  <system-out/>
  <testcase name="verifyLoggedInUserTest" time="0.000" classname="com.qa.hotspot.tests.HomePageTest">
    <skipped/>
  </testcase> <!-- verifyLoggedInUserTest -->
  <system-out/>
</testsuite> <!-- com.qa.hotspot.tests.HomePageTest -->
